app/models/purchase.py

```diff
@@ -103,7 +104,25 @@ def order_cart_items_from_vendor(vendor_id, date=None):
                             current_user.cart_items)
 
         order = Order(date, vendor_id)
-        # TODO: email vendor
+
+        vendor = User.query.get(vendor_id)
+        merchant_id = current_user.id
+        merchant = Merchant.query.get(merchant_id)
```

Comment:
Why can't we just use the User model for getting the merchant info (like we do for the vendor)? I am pretty sure it works either way, but perhaps it would be more efficient to just go with the User model since that avoids loading in an extra model.